spec/features/courses/requirement_spec.rb

```diff
@@ -1,21 +1,28 @@
 require 'rails_helper'
```

Comment:
Should this file be `requirements_spec.rb`? Just to keep things ~~standard~~consistent. ;)

Reply:
💯 good spot